app/services/license/license-service.js

```diff
@@ -0,0 +1,17 @@
+export default class AppsLicenseService extends $e.modules.ComponentBase {
+	getNamespace() {
+		return 'app/services/license';
+	}
+
+	isActive() {
```

Comment:
I'm assuming that you are planning to overwrite this from the pro, what will happen if the user won't update the pro plugin?